src/controllers/EventsController.php

```diff
@@ -222,7 +223,12 @@ public function postAction($request, $db) {
                     $users = $user_mapper->getUserById($request->user_id);
                     $thisUser = $users['users'][0];
 
-                    $commentText = filter_var($request->getParameter('comment'), FILTER_SANITIZE_STRING);
```

Comment:
Maybe we should consider this a web2 issue rather than an api issue. Should I restore the filter and open an issue for web2?